public/app/features/dashboard/components/GenAI/hooks.ts

```diff
@@ -46,6 +48,21 @@ export function useOpenAIStream(
   const [error, setError] = useState<Error>();
   const { error: notifyError } = useAppNotification();
 
+  const onError = useCallback(
+    (e: Error) => {
+      setStreamStatus(StreamStatus.IDLE);
+      setMessages([]);
+      setError(e);
+      notifyError(
+        'Failed to generate content using OpenAI',
+        `Please try again or if the problem persists, contact your organization admin.`
```

Comment:
This should read (just adding a comma):
`Please try again, or if the problem persists, contact your organization admin.`